### DISCUSSION TITLE
能否去掉开机时候的命令行窗口

### DISCUSSION THREAD

**Discussion by ctrlshiftm129** [Antonoko/Windrecorder#229] [Ideas 我有个想法]
每次开机会弹出来，很麻烦，而且不好看

**Comment by Antonoko** [comment]
谢谢反馈，已优化该特性，欢迎更新尝试

**Reply by ctrlshiftm129** [reply]
其实我感觉用任务计划程序，或者用个exe调用是比vbs更好的实现方式，不会有黑窗口一闪而过

**Reply by Antonoko** [reply]
考虑到运行过程需要保持透明和尽可能简单，如果只是非常快地一闪而过我觉得也还是可以接受捏。另一个体验的问题是python初始化太慢了，在托盘有响应前，用户可能会认为程序可能闪退、没能正确执行、从而多次执行启动
